deployments/operator/samples/deviceplugin_v1_gpudeviceplugin.yaml

```diff
@@ -4,3 +4,8 @@ metadata:
   name: gpudeviceplugin-sample
 spec:
   image: intel/intel-gpu-plugin:0.18.0
+  initImage: intel/intel-gpu-initcontainer:0.19.0
+  sharedDevNum: 10
+  logLevel: 4
+  nodeSelector:
+    accelerator: hd-graphics
```

Comment:
`nodeSelector`  (in all samples) we may need to keep commented out because these are not in the clusters by default.

Reply:
These samples are supposed to be just an illustration for what could be the CRs' content and they appear in the hub's UI, e.g. click "view YAML example" at https://operatorhub.io/operator/akka-cluster-operator. Unfortunately any comments get wiped out from them (in the YAML-to-JSON transformation phase) and don't show up in the UI.

Also the `kustomization.yaml` file in the `samples` directory lists all the samples as its resources so that operator-sdk can find and include them in the generated CSV. I guess adding overlays for such "base" doesn't make much sense because it implies running all the possible plugins in a cluster.

Reply:
@rojkov OK, I did not think of this operatorhub.io aspect. My vision is that one day our CRDs would depend on `NodeFeatureDiscovery`  and we'd rely on nodeselector labels detected by NFD. This is probably good now but we should look to have something NFD labeled here...

Reply:
I see. I've changed the labels a bit to suggest they are supposed to be used with NFD together.